go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/containernetworking/cni v0.7.2-0.20190904153231-83439463f784
 	github.com/containernetworking/plugins v0.8.2
 	github.com/containers/buildah v1.11.5
-	github.com/containers/conmon v2.0.2+incompatible // indirect
+	github.com/containers/common v0.0.2
```

Comment:
Is this version right?  I would have suspect v2.0.3?

Reply:
fixed. stays v2.0.2 the same as master.

Reply:
OK, but you've v0.0.2 here, not v2.0.2...

Reply:
conmon is v2.0.2 but common is v0.0.2

Reply:
TYVM @QiWang19 !  My eyes would never have spotted that with the particular font that GitHub uses by default for me.